app/models/proposal.rb

```diff
@@ -12,10 +12,20 @@
 
 class Proposal < ApplicationRecord
   enum status: { registered: 0, accepted: 1, rejected: 2 }
+  STATUS_MESSAGE = {
+    'registered' => 'エントリー済み',
+    'accepted' => '採択',
+    'rejected' => '不採択'
+  }.freeze
 
   belongs_to :talk
 
   def speakers
     talk.speakers
   end
+
+  def status_message
+    return 'エントリー済み' unless talk.conference.cfp_result_visible
```

Comment:
[IMO] STATUS MESSAGEがあるのでそちらの値を使いたい

```suggestion
    return STATUS_MESSAGE['registered'] unless talk.conference.cfp_result_visible
```